app/models/sub_section.rb

```diff
@@ -0,0 +1,70 @@
+class SubSection
+  attr_reader :sub_section_slug
+
+  delegate :title, to: :tag_from_content_api
+
+  def initialize(sub_section_slug)
+    @sub_section_slug = sub_section_slug
+  end
+
+  def lists
+    @lists ||= curated_lists || fallback_list_of_uncurated_links
+  end
+
+  def a_to_z?
+    lists
+    @a_to_z
+  end
+
+  def exists?
+    tag_from_content_api.present?
+  end
+
+private
+
+  def curated_lists
+    return unless subsection_from_content_store.details && subsection_from_content_store.details.groups
+
+    @a_to_z = false
+
+    lists = subsection_from_content_store.details.groups.map do |group|
+      links = group.contents.map do |url|
+        tagged_items_from_content_api.find { |artefact| artefact.id == url }
+      end
+
+      OpenStruct.new(name: group.name, links: links)
+    end
+  end
+
+  def fallback_list_of_uncurated_links
+    @a_to_z = true
+
+    [OpenStruct.new(
+      name: 'A&#8202;to&#8202;Z',
```

Comment:
It would be good to keep the comment about what the `&#8202;` entity is next to this as it isn't immediately obvious